deepspeed/runtime/zero/parameter_offload.py

```diff
@@ -145,6 +145,16 @@ def __init__(
             module.ds_inflight_param_registry = InflightParamRegistry()
         self.__inflight_param_registry = module.ds_inflight_param_registry
 
+        self.fast_fetch_for_leaf_module = False
```

Comment:
@inkcherry, since this great feature accelerates both fetch and release, I think it would be helpful for the naming to reflect this. The current naming suggests that only fetch is accelerated. So, can we consider more general naming, such as below?

```suggestion
        self.fast_sharding_for_leaf_module = False
```

Reply:
thanks~ yes, I’ve updated the naming .